cylc/flow/taskdef.py

```diff
@@ -361,3 +358,6 @@ def is_parentless(self, point):
             or all(x < self.start_point for x in parent_points)
             or self.has_only_abs_triggers(point)
         )
+
+    def __repr__(self) -> str:
```

Comment:
```suggestion
    def __repr__(self) -> str:
        """
        Examples:
            >>> from types import SimpleNamespace
            >>> x = SimpleNamespace(name="foo")
            >>> TaskDef.__repr__(x)
            '<TaskDef foo>'
        """

or add `# pragma nocover`.

IMO any test is essentially testing f-strings, so not worth it.

Reply:
Or just accept a miniscule drop in coverage?

Reply:
Whatever

Reply:
Or, crazy idea, test it!

Reply:
Given the content that's perilously close to testing a builtin.

Reply:
But I don't want to test it 😥